coda-src/venus/comm.cc

```diff
@@ -673,11 +672,11 @@ void DoProbes(int HowMany, struct in_addr *Hosts)
 
 void MultiBind(int HowMany, struct in_addr *Hosts, connent **Connections)
 {
-    if (LogLevel >= 1) {
+    if (GetLogLevel() >= 1) {
         dprint("MultiBind: HowMany = %d\n\tHosts = [ ", HowMany);
         for (int i = 0; i < HowMany; i++)
-            fprintf(logFile, "%s ", inet_ntoa(Hosts[i]));
-        fprintf(logFile, "]\n");
+            fprintf(GetLogFile(), "%s ", inet_ntoa(Hosts[i]));
+        fprintf(GetLogFile(), "]\n");
```

Comment:
The change from `logFile` to `GetLogFile()` is just to avoid using `extern` variable and contain the control of `logFile`'s value within its subsystem. I didn't mean to change the logging functionality.

I agree with all your logging concerns and I'd be happy to see (or even implement) the logging subsystem rework. Since this PR is meant to be about creating a key/value store for Venus configuration parameters. Shouldn't we probably deffer this concerns somewhere else. Possibly filing an issue out of this discussion.